extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -309,7 +309,7 @@
 $messages['es'] = [
 	'communitypage-title' => 'Comunidad',
 	'communitypage-header-welcome' => '¡Bienvenido a la comunidad de {{SITENAME}}!',
-	'communitypage-header-join-button' => 'Participar ahora',
+	'communitypage-header-join-button' => 'Participa ahora',
 	'communitypage-admin-welcome-message' => 'Comparte tu conocimiento editando páginas, subiendo imágenes y añadiendo nuevo contenido.',
 	'communitypage-admin-group-name' => 'Administradores',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Administradores" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)